testsuite/OMSimulator/snapshot.py

```diff
@@ -106,6 +106,15 @@
 ## 			</ssv:Parameters>
 ## 		</ssv:ParameterSet>
 ## 	</oms:ssv_file>
+## 	<oms:signalFilter_file name="resources/signalFilter.xml">
```

Comment:
Please change it to `<oms:file name="resources/signalFilter.xml">`

Reply:
we can fix this in another PR